Test_Automation_Framework/Epam_TestAutomation_API_Tests/appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "BiblesBaseUrl": "https://api.scripture.api.bible",
+  "PhonesBaseUrl": "https://api.restful-api.dev",
+  "ApiKey": "8f5f64223e4f3d5a278900be66a1aa8b"
```

Comment:
Why API key is in the repo????? I marked it bold and underlined in HW clarification. Please be more attentive to details!